tasks/site.yml

```diff
@@ -137,6 +137,32 @@
     mode: '0644'
   tags: [ 'role::checkmk_server:multisite' ]
 
+- name: Wait for the site to be started
+  wait_for:
+    path: '{{ checkmk_server__site_home }}/{{ checkmk_server__multisite_config_path }}/wato/users.mk'
+    timeout: 60
+
+- name: Read local multisite users definition
+  command: 'sed "1,4d" {{ checkmk_server__site_home }}/{{ checkmk_server__multisite_config_path }}/wato/users.mk'
```

Comment:
Relying on fixed lines looks a bit fragile. Maybe something like this `sed -e "1,/^multisite_users\s*=/d"` would be more robust?